lib/radbeacon/le_scanner.rb

```diff
@@ -1,10 +1,11 @@
 module Radbeacon
 
   class LeScanner
-    attr_accessor :duration
+    attr_accessor :duration, :options
 
     def initialize(duration = 5)
       @duration = duration
+      @options = {}
```

Comment:
Oh that's neat.  I created an issue (#4) to track this and plan to implement it once we update the Pis running zarkorr to Ruby 2.0.  For now, I changed the option to `enable_hcitool_duration` which is hopefully more clear.